composeml/label_maker.py

```diff
@@ -106,16 +109,17 @@ def df_to_labels(df):
             labels.index = labels.index.astype('datetime64[ns]')
             return labels.to_frame()
 
+        df = self._preprocess(df)
+        labels = df.groupby(self.target_entity)
+        progress_bar = None
+
         if verbose:
             bar_format = "Elapsed: {elapsed} | Remaining: {remaining} | "
             bar_format += "Progress: {l_bar}{bar}| "
             bar_format += self.target_entity + ": {n}/{total} "
-            tqdm.pandas(bar_format=bar_format, ncols=90)
+            progress_bar = tqdm(total=labels.ngroups, bar_format=bar_format, ncols=90)
```

Comment:
do you need to set `ncols` yourself here? we don't do it in Featuretools, I don't have any issues I am aware of

Reply:
When I don't use `ncols`, the progress bar stretches across the screen. Will send you examples.